### ISSUE TITLE
cannot reconcile overleaf and local versions of files, refusing to copy overleaf version

### ISSUE THREAD

**Issue by outs1der** [showyourwork/showyourwork#496]
having solved the access issue I cannot for the life of me get changes on overleaf propagated to the local version. 

Every time I run `showyourwork build` it reports 

> Local file changed since the last Overleaf sync: ms.tex. Refusing to overwrite with Overleaf version. Please see the docs for details on how to resolve this.

I have copied the local files back to overleaf and still get the same error. Timestamp on the local file is < latest edit on the Overleaf version. They are otherwise identical. Why does syw think the local one is newer/has changes?

**Comment by HealthyPear** [comment]
I guess you already saw this section of the docs?
https://show-your.work/en/latest/overleaf/#managing-conflicts

It think here there is a mismatch in the commits between your remote repo and Overleaf (as the latter also is connected to your repo and commits itself when you edit from there).
Depending if you commit and pushed your local changes you need to deal with the conflict in slightly different manner

**Comment by outs1der** [comment]
yeah I read through that but could not really parse the correct steps to a solution

to my mind, copying the local file to remote OR remote to local should fix the problem, but even when they are identical I get the same error. 

how to diagnose/fix a mismatch in the commits?

**Comment by outs1der** [comment]
It occurred to me I could just replace the linked Overleaf project, so I followed the instructions here https://show-your.work/en/latest/overleaf/#existing-git-repository:
* created new (blank) repository (Overleaf includes a `main.tex` file)
* copied over the `ms.tex` and `bib.bib` file from the local repo by dragging into the file pane on the LHS of the Overleaf interface
* added the Overleaf project ID to `showyourwork.yml`
* sync with `showyourwork build`

but it *still* reports 
> Local file changed since the last Overleaf sync: ms.tex. Refusing to overwrite with Overleaf version. Please see the docs for details on how to resolve this.

I do have Dropbox syncing on my Overleaf account, don't know if that might be a factor

**Comment by HealthyPear** [comment]
Hello,

I recently had the same issue in one of my project and I fixed it by following the documentation (in particular the part which describes what to do if one accidentally also pushes the changes to the GitHub repo, not only stages or commits locally a change that is not on Overleaf) and I managed to fix the conflict.

Given this I will (tentatively) close this issue.
